### DISCUSSION TITLE
X86_64 Mini OpenWrt 无法编译已经两天了

### DISCUSSION THREAD

**Discussion by geekbb** [haiibo/OpenWrt#192] [General]
![CleanShot 2024-01-30 at 09 08 04@2x](https://github.com/haiibo/OpenWrt/assets/46196358/57632dc1-defd-42df-8e5a-2e536f9c5dc1)

**Comment by tearsful** [comment]
@geekbb [https://github.com/xiaorouji/openwrt-passwall-packages/issues/117](url) passwall那里缺少clang，你可以在工作流里加上安装clang的命令！
<img width="449" alt="clang" src="https://github.com/haiibo/OpenWrt/assets/110714634/90f18a2e-2032-4fc6-a27e-7d35d7ee06e6">

**Reply by geekbb** [reply]
![CleanShot 2024-02-02 at 08 18 22@2x](https://github.com/haiibo/OpenWrt/assets/46196358/8bb1d16d-2ab0-4d04-9e54-cf79c72df36f)
![CleanShot 2024-02-02 at 08 20 32@2x](https://github.com/haiibo/OpenWrt/assets/46196358/d83296e0-9ee0-4327-930c-3766b07d22fa)
添加了还是失败

**Reply by tearsful** [reply]
这个信息已经不是passwall报错了。建议同步一下再来添加

**Reply by geekbb** [reply]
不知道您有没有注意到，这个库(haiibo/OpenWrt)本库X86也编译出错了🤣

**Reply by tearsful** [reply]
本地库出错应该也是没改，我fork了大佬的库后做了一些增加和修改。设置了每天定时编译。这两天一直是正常的。

**Reply by tearsful** [reply]
@geekbb 我看了一下你最后一个编译的信息，你用的是x86-mini的配置，信息报错是iperf3，将config配置里的 CONFIG_PACKAGE_iperf3=y去掉就应该没啥问题了。
<img width="476" alt="iperf3" src="https://github.com/haiibo/OpenWrt/assets/110714634/4848f22e-7210-434d-8a56-a5a956f37c0d">

**Reply by geekbb** [reply]
感谢！感谢！已经成功了
